component/core/src/main/java/org/exoplatform/social/core/jpa/storage/dao/jpa/SpaceDAO.java

```diff
@@ -389,6 +389,11 @@ private void buildPredicates(XSpaceFilter spaceFilter,
       parameterNames.add(PARAM_KEYWORD);
     }
 
+    if (StringUtils.isNotBlank(spaceFilter.getRemoteId())) {
+      suffixes.add("UserId");
+      parameterNames.add(PARAM_USER_ID);
+    }
```

Comment:
Thanks, please remove this line:
https://github.com/Meeds-io/social/blob/e4288e948d479c2dca9261fc2101be3eee349e80/component/core/src/main/java/org/exoplatform/social/core/jpa/storage/dao/jpa/SpaceDAO.java#L312